kolibri/plugins/coach/assets/src/views/plan/assignments/AssignmentDetailsModal.vue

```diff
@@ -318,6 +317,9 @@
       handleSubmitTitleFailure() {
         this.formIsSubmitted = false;
         this.showTitleError = true;
+        this.$refs.titleField.focus();
+        // Scroll to the title field in case focus() didn't do that immediately
+        this.$refs.titleField.$el.scrollIntoView({ behavior: 'smooth' });
```

Comment:
non-blocking: my only suggestion for here is that maybe we just scroll to the top of the page, since the app bar is covering the field with the default scroll behavior?

https://github.com/learningequality/kolibri/assets/17235236/9b9edd23-bfb1-4a2a-92f2-8e80c16cf461